src/core/Path.ts

```diff
@@ -112,6 +112,12 @@ export class Vector implements Coordinate {
     return rtn;
   }
 
+  rotate(angle: number): Vector {
```

Comment:
Please write test cases for this function.

Also, please check out `Coordinate.euclideanRotation`, maybe you want to use that function. Both approaches are okay.